conf/tuos_stanage.config

```diff
@@ -30,10 +30,8 @@ process {
 
   // scheduler
 
-  executor = 'sge'
+  executor = 'slurm'
   penv = 'smp'
```

Comment:
```suggestion

```

I don't think a `penv` should break anything (with slurm) but it's not used either.